cd_win.c

```diff
@@ -342,16 +348,6 @@ static void CD_f (void)
 		return;
 	}
 
-	if (Q_strcasecmp(command, "info") == 0)
-	{
-		Con_Printf("%u tracks\n", maxTrack);
```

Comment:
I don't know why this was removed, playing, and pausing music is a core part of the application

Reply:
test